modules/generate/generate.go

```diff
@@ -34,6 +35,64 @@ func GetRandomString(n int) (string, error) {
 	return string(buffer), nil
 }
 
+// GetRandomPassword generates a random password
+func GetRandomPassword(n int) (string, error) {
+	const lower = "abcdefghijklmnopqrstuvwxyz"
+	const upper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	const digit = "0123456789"
+	const spec = "!@#$%^&*()_-+=[]'\";:/?.>,<`~"
```

Comment:
Still some special characters here; the commit you've just pushed is for another part of the code.